applications/crossbar/src/modules/cb_token_auth.erl

```diff
@@ -209,10 +209,16 @@ is_expired(Context, JObj) ->
     AccountId = wh_json:get_value(<<"account_id">>, JObj),
     case wh_util:is_account_expired(AccountId) of
         'false' -> check_as(Context, JObj);
-        'true' ->
+        {'true', Expired} ->
```

Comment:
Did you forget to commit a change to `wh_util`? `wh_util:is_account_expired/1` returns a `boolean()` only. I hope that's the case versus this being an untested changeset!